api/user.go

```diff
@@ -67,6 +67,12 @@ func (create UserCreate) Validate() error {
 	if len(create.Username) > 32 {
 		return fmt.Errorf("username is too long, maximum length is 32")
 	}
+	if len(create.Password) < 6 {
+		return fmt.Errorf("password is too short, minimum length is 6")
+	}
+	if len(create.Password) > 64 {
```

Comment:
BTW, the password is generated random hash, so we don't have to check its maximum.

Reply:
I don't know much about security, but it seems like it could be a point of attack if we don't restrict the max length. E.g., `long password denial of service`.
I think we can set a larger value, such as 512, to ensure that is enough for almost all users. WDYT?

Reply:
`512` LGTM